src/Auth0/Login/Traits/Auth0UserTrait.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Auth0\Login\Traits;
+
+trait Auth0UserTrait {
+
+    /**
+     * @var array
+     */
+    protected $userInfo;
+
+    /**
+     * @var string
+     */
+    protected $accessToken;
+
+    /**
+     * Get the unique identifier for the user.
+     *
+     * @return string|null
+     */
+    public function getAuthIdentifier() : ?string
+    {
+        return $this->userInfo['sub'] ?? null;
+    }
+
+    /**
+     * Get id field name.
+     *
+     * @return string
+     */
+    public function getAuthIdentifierName() : string
+    {
+        return 'id';
```

Comment:
This is confusing to me ... should this continue to be `"id"` as it was before? Seems like it should be `"sub"` but maybe I'm not understanding the purpose.